recipe/meta.yaml

```diff
@@ -5,12 +5,12 @@ package:
     version: {{ version }}
 
 source:
-    fn: Fiona-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/F/Fiona/Fiona-{{ version }}.tar.gz
-    md5: f6b70e1a30fc8db597c360e375d186de
+    fn: Fiona-{{ version }}.post1.tar.gz
+    url: https://pypi.python.org/packages/source/F/Fiona/Fiona-{{ version }}.post1.tar.gz
```

Comment:
> PyPI forces it upon you - you can't override a bad upload.

The bad upload case is, IMO, a PyPI bug. (Easy to fix BTW. The release
should be official only after some sort of file integrity test is
performed.) But people are using post releases for minor fixes and that
drives me nuts ;-)